charts/cluster-security/values.yaml

```diff
@@ -0,0 +1,2 @@
+baselineNamespaces: [apps, default]
+privilegedNamespaces: [armo-system, cluster-services, kube-node-lease, kube-public, kube-system, kubescape-host-scanner, monitoring]
```

Comment:
nit: consider using the YAML syntax here

Reply:
I copied the same syntax from cluster-secrets/values.yaml.
So something in the form of

```namespaces:
 - apps
 - default
```
 would be more appropriate?